custom_components/waste_collection_schedule/waste_collection_schedule/source/vestfor_dk.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+import requests as request
+import json
+import logging
+from waste_collection_schedule import Collection
+
+_LOGGER = logging.getLogger(__name__)
+
+TITLE = "Vestforbrænding" # Title will show up in README.md and info.md
+DESCRIPTION = "Source for Vestforbrændning collection"  # Describe your source
+URL = "https://selvbetjening.vestfor.dk/"  # Insert url to service homepage. URL will show up in README.md and info.md
+TEST_CASES = {  # Insert arguments for test cases to be used by test_sources.py script
+    "Home": {"streetName": "Kløvertoften", "number": "61", "zipCode": "2740"}
+}
+
+API_URL = "https://selvbetjening.vestfor.dk/Adresse/ToemmeDates"
+ICON_MAP = {   # Optional: Dict of waste types and suitable mdi icons
+    "Haveaffald": "mdi:leaf",
+    "Storskrald": "mdi:recycle",
+    "Mad/Rest affald": "mdi:food",
+    "Pap": "mdi:archive",
+    "Papir/Plast \u0026 MDK": "mdi:bottle-soda",
+    "Metal/Glas affald": "mdi:wrench",
+    "Juletræer": "mdi:pine-tree",
+}
+
+ADRESS_LOOKUP_URL = "https://selvbetjening.vestfor.dk/Adresse/AddressByName"
+
+class Source:
+    def __init__(self, streetName, number, zipCode):
+        self._streetName = streetName
+        self._number = number
+        self._zipCode = zipCode
+
+    def fetch(self):
+        entries = []  # List that holds collection schedule
+
+        term = self._streetName + " " + self._number + ", " + self._zipCode
+        term = request.utils.quote(term)
+
+        _LOGGER.info("Fetching addressId from Vestforbrændning: " + term)
+        queryAddress = ADRESS_LOOKUP_URL + "?term=" + term + "&numberOfResults=10"
+        addressResponse = request.get(queryAddress)
+        addresses = json.loads(addressResponse.text)
+
+        if len(addresses) == 0:
+            return entries
```

Comment:
Please raise an issue as this will be shown in the logs and just returning an empty result will not tell the user that something went wrong (and what).